components/sup/src/event.rs

```diff
@@ -16,6 +16,8 @@ mod stream;
 mod types;
 
 pub(crate) use self::types::ServiceMetadata;
+use prost_types::Duration as prost_duration;
```

Comment:
Rust types are generally represented in pascal case. I think `ProstDuration` would be better.